lib/components/art/ArtDetailWidget.dart

```diff
@@ -11,26 +14,89 @@ class ArtDetailWidget extends StatefulWidget {
 }
 
 class _ArtDetailWidgetState extends State<ArtDetailWidget> {
-  @override
   Widget build(BuildContext context) {
     return Container(
       color: Colors.white,
-      child: Center(
-        child: Column(
-          mainAxisAlignment: MainAxisAlignment.center,
-          children: <Widget>[
-            Text(
-              widget.data.title,
-              style: TextStyle(
-                  fontSize: 36.0, color: Theme.of(context).primaryColor),
+      child: Column(
+        mainAxisAlignment: MainAxisAlignment.start,
+        children: <Widget>[
+          Card(
+            shape: RoundedRectangleBorder(
+              borderRadius: BorderRadius.circular(25.0),
+            ),
+            child: Container(
+              width: MediaQuery.of(context).size.width,
```

Comment:
I wouldn't use the size of the screen here just because of the difference between mobile view and desktop view. This may work on mobile but you should limit it on tablet and desktop or else it will stretch across the entire screen.

To fix this maybe do a multiplier depending on screen size (e.g. 80%) or make a fixed size for the image. 

Or what i suggest base it on a certain aspect ratio like 16:9.  So for a height for 250 -> use a 450 fixed width. https://calculateaspectratio.com/ round up a bit from this calculator